addons/plotting/functions/fnc_compileFormatters.sqf

```diff
@@ -0,0 +1,52 @@
+#include "script_component.hpp"
+/*
+ * Authors: Timi007
+ * Compiles formatters from config and saves them in missionNamespace.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call zen_plotting_fnc_compileFormatters
+ *
+ * Public: No
+ */
+
+private _fnc_getFormatters = {
+    params ["_cfgFormatters"];
+
+    private _formatters = [];
+    {
+        private _entryConfig = _x;
+
+        private _formatterName = configName _entryConfig;
+
+        private _formatterString = getText (_entryConfig >> "formatter");
+        if (_formatterString isNotEqualTo "") then {
+            _formatterString = "params ['_value']; " + _formatterString;
```

Comment:
Unnecessary, just pass the value in `_this` and then, `format ["%1 m", _this]` for example.